src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/ApplicationTest.kt

```diff
@@ -2378,7 +2378,7 @@ class ApplicationTest : IntegrationTestBase() {
               CaseDetailFactory().produce(),
             )
 
-            val responseStatuses = mutableListOf<HttpStatus>()
+            var responseStatuses = listOf<HttpStatusCode>()
```

Comment:
i'm not sure why this has been changed? Can we revert it?

Reply:
TODO